nilearn/_utils/extmath.py

```diff
@@ -75,3 +79,61 @@ def is_spd(M, decimal=15, verbose=1):
     if not ispd and verbose > 0:
         print("matrix has a negative eigenvalue: %.3f" % eigvalsh.min())
     return ispd
+
+
+def check_threshold(threshold, data, percentile_calculate, name='threshold'):
```

Comment:
As discussed with @GaelVaroquaux . this function will be symbolled as private and will move to new file ```param_validation.py``` in _utils.
@AlexandreAbraham @lesteve